<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewer</class>
 <widget class="QWidget" name="viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>820</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VimbaLive</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <item>
        <widget class="QGroupBox" name="ImageGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Live Image</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0">
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
             <item>
              <layout class="QVBoxLayout" name="LeftPlotLayout">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
              </layout>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QWidget" name="ImageWidget" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>500</width>
                    <height>0</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_13"/>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="TopPlotLayout"/>
                </item>
                <item>
                 <widget class="PyDMImageView" name="ImageView" native="true">
                  <property name="toolTip">
                   <string/>
                  </property>
                  <property name="maxRedrawRate" stdset="0">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_3">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="CursorLabel">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="MiscLabel">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="StatsWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Centroid</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="3" column="1">
                 <widget class="PyDMLabel" name="OrientationLabel">
                  <property name="toolTip">
                   <string/>
                  </property>
                  <property name="precision" stdset="0">
                   <number>3</number>
                  </property>
                  <property name="precisionFromPV" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="PyDMLabel" name="SemiminorLabel">
                  <property name="toolTip">
                   <string/>
                  </property>
                  <property name="precision" stdset="0">
                   <number>3</number>
                  </property>
                  <property name="precisionFromPV" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>Semimajor length</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="PyDMLabel" name="CentroidLabel">
                  <property name="toolTip">
                   <string/>
                  </property>
                  <property name="precision" stdset="0">
                   <number>2</number>
                  </property>
                  <property name="precisionFromPV" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>Orientation</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Semiminor length</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="PyDMLabel" name="SemimajorLabel">
                  <property name="toolTip">
                   <string/>
                  </property>
                  <property name="precision" stdset="0">
                   <number>3</number>
                  </property>
                  <property name="precisionFromPV" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Center:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Gaussian fit (1/e^2 radius)</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="2">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>w_x (mm):</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QFrame" name="frame_2">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_12">
                   <item>
                    <widget class="PyDMLabel" name="wymmLabel">
                     <property name="toolTip">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>w_y (mm):</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>w_y (pixel):</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="PyDMLabel" name="wxPixelLabel">
                  <property name="toolTip">
                   <string/>
                  </property>
                  <property name="precisionFromPV" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>w_x (pixel):</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="PyDMLabel" name="wyPixelLabel">
                  <property name="toolTip">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QFrame" name="frame">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="PyDMLabel" name="wxmmLabel">
                     <property name="toolTip">
                      <string/>
                     </property>
                     <property name="precisionFromPV" stdset="0">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="RefGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Reference</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0,0">
          <item>
           <widget class="QWidget" name="RefWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_16"/>
           </widget>
          </item>
          <item>
           <widget class="PyDMImageView" name="RefView" native="true">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="RefCursorLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="RefMiscLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_9">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_17">
                <item>
                 <widget class="QLabel" name="label_21">
                  <property name="text">
                   <string>R^2:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PyDMLabel" name="DetLabel">
                  <property name="toolTip">
                   <string/>
                  </property>
                  <property name="precision" stdset="0">
                   <number>5</number>
                  </property>
                  <property name="precisionFromPV" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMImageView</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.image</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
